### PR TITLE
Queue

### PR DESCRIPTION
**Pull request by TheStaplergun**
Here's the stuff!

### CODE REVIEW

**Comment by TheStaplergun** [comment]
Any changes you want on this?

**Comment by Mengmoshu** [comment]
No idea, I keep forgetting about it when I actually have time to look it over. I can try to give it a look tomorrow.

**Comment by TheStaplergun** [comment]
Aw yeah!

**Comment by Mengmoshu** [comment]
It's now "tomorrow", and I've had a chance to look this over. I'm kinda having trouble *understanding* it, but it doesn't look alien. I think I'm comfortable merging it, especially since getting this merged in will make it part of the version for Factorio 0.18.

Am I reading right that there's a `rmm` table? (Two Ms)

File: control.lua [hunk 1]
```diff
@@ -1,58 +1,4 @@
...
-        if myDebug == true then
-            event.created_entity.surface.print("new RRM inserted to global.RRMs")
-        end
-    end
-end
-
-script.on_event(defines.events.on_built_entity, builtEntity)
-script.on_event(defines.events.on_robot_built_entity, builtEntity)
+require("scripts.machine")
+require("scripts.rmm-queue")
```
Comment (Mengmoshu) [inline control.lua:3]: Here's an example of what I commented about, though I guess I remembered the usage of rmm wrong. Kerning issue?

**Pull request merged**